recipes/serde/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "serde" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/rossmacarthur/serde/archive/{{ version }}.tar.gz"
+  sha256: 738dcd50ef7b28f5be7bfc3e49762aba1d3d3ae71786375ce74163ff42f93c6b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - isodate >=0.6
```

[This version](https://github.com/rossmacarthur/serde/blob/71662749f5eca79e801df6642987cfc912c1c661/setup.py#L46) has a dependency on `six` as wel;:
```suggestion
    - isodate >=0.6
    - six >=1.13.0
```

---

great thanks!